### PR TITLE
[Bulk] Validation: Fix various issues in azuredocssdkjava

### PR DESCRIPTION
**Pull request by reemsabaw1**
This PR fixes 83 issues in azure docs sdk java.

### CODE REVIEW

File: docs-ref-services/latest/digitaltwins-core-readme.md [hunk 1]
```diff
@@ -1,14 +1,13 @@
 ---
 title: Azure IoT Digital Twins client library for Java
+description: This library provides access to the Azure Digital Twins service for managing twins, models, relationships, etc.
```
Comment (LSchaufMS) [inline docs-ref-services/latest/digitaltwins-core-readme.md:3]: Change "etc." to " and more."

File: docs-ref-services/latest/hdinsight.md [hunk 2]
```diff
@@ -59,7 +59,7 @@ You will also need to add the following dependencies to your pom.xml:
 The SDK first needs to be authenticated with your Azure subscription.  Follow the example below to create a service principal and use it to authenticate. After this is done, you will have an instance of an `HDInsightManagementClientImpl`, which contains many methods (outlined in below sections) that can be used to perform management operations.
 
 > [!NOTE]
-> There are other ways to authenticate besides the below example that could potentially be better suited for your needs. All methods are outlined here: [Authenticate with the Azure management libraries for Java](https://docs.microsoft.com/en-us/java/azure/java-sdk-azure-authenticate?view=azure-java-stable)
+> There are other ways to authenticate besides the below example that could potentially be better suited for your needs. All methods are outlined here: [Authenticate with the Azure management libraries for Java](/java/azure/java-sdk-azure-authenticate?view=azure-java-stable&preserve-view=true)
```
Comment (LSchaufMS) [inline docs-ref-services/latest/hdinsight.md:62]: I think that this changed link does need to include the /en-us/: "/en-us/java/azure/java-sdk-azure-authenticate?view=azure-java-stable"

File: docs-ref-services/latest/hdinsight.md [hunk 3]
```diff
@@ -353,7 +353,7 @@ The HDInsight Management SDK can also be used to manage monitoring on your clust
 ### Enable OMS Monitoring
 
 > [!NOTE]
-> To enable OMS Monitoring, you must have an existing Log Analytics workspace. If you have not already created one, you can learn how to do that here: [Create a Log Analytics workspace in the Azure portal](https://docs.microsoft.com/en-us/azure/log-analytics/log-analytics-quick-create-workspace).
+> To enable OMS Monitoring, you must have an existing Log Analytics workspace. If you have not already created one, you can learn how to do that here: [Create a Log Analytics workspace in the Azure portal](/azure/log-analytics/log-analytics-quick-create-workspace).
```
Comment (LSchaufMS) [inline docs-ref-services/latest/hdinsight.md:356]: I think that this changed link does need to include the /en-us/ at the front: "/en-us/azure/log-analytics/log-analytics-quick-create-workspace"

File: docs-ref-services/latest/hdinsight.md [hunk 4]
```diff
@@ -381,7 +381,7 @@ client.extensions().disableMonitoring("<Resource Group Name>", "<Cluster Name>")
 
 HDInsight provides a configuration method called script actions that invokes custom scripts to customize the cluster.
 > [!NOTE]
-> More information on how to use script actions can be found here: [Customize Linux-based HDInsight clusters using script actions](https://docs.microsoft.com/en-us/azure/hdinsight/hdinsight-hadoop-customize-cluster-linux)
+> More information on how to use script actions can be found here: [Customize Linux-based HDInsight clusters using script actions](/azure/hdinsight/hdinsight-hadoop-customize-cluster-linux)
```
Comment (LSchaufMS) [inline docs-ref-services/latest/hdinsight.md:384]: I think that this changed link does need to include the /en-us/ at the front: "/en-us/azure/hdinsight/hdinsight-hadoop-customize-cluster-linux"

**Review by LSchaufMS — CHANGES_REQUESTED** [review]
A few minor changes, and some questions about the "en-us/" segment on relative links.

**Comment by reemsabaw1** [comment]
@g2vinay @sima-zhu @johngallardo @vcolin7 
@conniey 


The Content & Learning team is fixing build validation errors on docs.microsoft.com for the rest of H2. This will eliminate potential accessibility, security, and usability issues. This PR includes only build validation fixes and does not change other content. Please review within five business days and merge or comment in the PR with any changes you’d like to see. Thank you!

**Comment by reemsabaw1** [comment]
@g2vinay @sima-zhu @johngallardo  @vcolin7 
@conniey 

The Content & Learning team is fixing build validation errors on docs.microsoft.com for the rest of H2. This will eliminate potential accessibility, security, and usability issues. This PR includes only build validation fixes and does not change other content. Please review within five business days and merge or comment in the PR with any changes you’d like to see. Thank you!

**Pull request closed without merging**